### ISSUE TITLE
Tab component with angular 9 ignores tab layouts

### ISSUE THREAD

**Issue by joachim-heck** [syncfusion/ej2-angular-ui-components#98]
Dear syncfusion team, after upgrading to angular 9 it seems that the tab component is still broken and doesn't show up at all... you can reproduce the error by simply upgrading the sample from stackblitz from: https://ej2.syncfusion.com/angular/demos/#/material/tab/default 

Angular 8:
![image](https://user-images.githubusercontent.com/49340614/76929090-9121d580-68e3-11ea-896d-a4b300f9fd55.png)

Angular 9:
![image](https://user-images.githubusercontent.com/49340614/76929022-646dbe00-68e3-11ea-9e76-097e557d0ecf.png)

**Comment by Balasubramanian-Sattanathan** [comment]
Hi Joachim heck,

We have validated and checked your reported scenario at our online demos. Since the tab is rendered properly when upgrading to version 9.0.0 like your shared screenshot. For your reference, we have attached the modified Stackblitz sample and also prepared an individual project with the Tab component in Angular 9.

Stackblitz sample: [](https://stackblitz.com/edit/angular-yxrwr5?file=app.component.ts) 
Individual sample: [](https://www.syncfusion.com/downloads/support/directtrac/269856/ze/tab_sample_ang_9900499629.zip) 

Kindly try the above samples and let us know if you have any concerns.

**Comment by Balasubramanian-Sattanathan** [comment]
Hi Joachim heck,

Greetings from Syncfusion Support.

We have validated and checked your reported scenario at our online demos. Since the tab is rendered properly when upgrading to version 9.0.0 like your shared screenshot. For your reference, we have attached the modified Stackblitz sample and also prepared an individual project with the Tab component in Angular 9.

Stackblitz sample: https://stackblitz.com/edit/angular-yxrwr5?file=app.component.ts
Individual sample: https://www.syncfusion.com/downloads/support/directtrac/269856/ze/tab_sample_ang_9900499629.zip 

Kindly try the above samples and let us know if you have any concerns.
